include/zephyr/bluetooth/conn.h

```diff
@@ -1244,6 +1354,20 @@ struct bt_conn_cb {
 				const struct bt_conn_le_path_loss_threshold_report *report);
 #endif /* CONFIG_BT_PATH_LOSS_MONITORING */
 
+#if defined(CONFIG_BT_SUBRATING)
+	/** @brief LE Subrate Changed event.
+	 *
+	 *  This callback notifies the application that the subrating parameters
+	 *  of the connection have changed. The new parameters can be obtained
+	 *  using @ref bt_conn_get_info if status is BT_HCI_ERR_SUCCESS.
+	 *
+	 *  @param conn   Connection object.
+	 *  @param status HCI status from LE Subrate Change event.
+	 */
+	void (*subrate_changed)(struct bt_conn *conn,
+				const uint8_t status);
```

Comment:
Did you consider giving a `bt_conn_le_subrating_info` in the callback as well? How useful is it to know that the subrating parameters changed without knowing the actual parameters? I expect most applications will want these parameters but I may be wrong.

Reply:
Modified to pass the changed parameters to the callback now, is it the implementation you had in mind too?

Reply:
Yes, something like this. Would be good to get feedback from some more people on the api questions